test/1_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_helper'
+
+class WelcomeControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    # blah
+  end
+
+  test '#index' do
+    # test goes here
+  end
+
+  # if I uncomment the 2nd test case within the same class (WelcomeControllerTest) then it would fail
```

Comment:
I get the following error:

```
rake aborted!
test_#index is already defined in WelcomeControllerTest
```

only if the below test case is uncommented.

I can't reproduce the error if 2 different controller test files have the same test case name.